### PR TITLE
feat: KEP-965 integrate geckoterminal api v2 for prochart

### PR DESCRIPTION
**Pull request by viet-nv**

### CODE REVIEW

File: src/services/geckoTermial.ts [hunk 1]
```diff
@@ -1,97 +1,95 @@
...
-      query: ({ network, poolAddress }) => ({
-        url: `/api/p1/${network}/pools/${poolAddress}?include=tokens`,
-      }),
-    }),
-
-    candelsticks: builder.query<CandleResponse, CandleParams>({
-      query: params => ({
-        url: '/contracts/1/candlesticks.json',
-        params,
+    ohlcv: builder.query<CandleResponse, CandleParams>({
```
Comment (nguyenhoaidanh) [inline src/services/geckoTermial.ts:78]: chấm hỏi

File: src/components/TradingViewChart/datafeed.tsx [hunk 2]
```diff
@@ -123,25 +122,24 @@ export const useDatafeed = (poolDetail: PoolResponse, tokenId: string) => {
...
-              time: new Date(item.dt).getTime(),
-              open: item.o,
-              high: item.h,
-              close: item.c,
-              low: item.l,
-              volume: item.v,
-            }))[0],
-          )
+          const bars =
+            data?.data?.data.attributes.ohlcv_list.map((item: any) => ({
```
Comment (nguyenhoaidanh) [inline src/components/TradingViewChart/datafeed.tsx:134]: duplicate, let's make a util func

File: src/services/geckoTermial.ts (hunk 1 above)
Comment (viet-nv) [reply src/services/geckoTermial.ts]: open high low close volume

File: src/components/TradingViewChart/datafeed.tsx (hunk 2 above)
Comment (viet-nv) [reply src/components/TradingViewChart/datafeed.tsx]: transform response from api

**Comment by namgold** [comment]
please use `prochart` flag from ks-setting configurations by `useKyberSwapConfig()`

**Comment by viet-nv** [comment]
> please use `prochart` flag from ks-setting configurations by `useKyberSwapConfig()`

I suggest to remove it from ks-settings

**Review by nguyenhoaidanh — APPROVED** [review]

**Pull request merged**